grunt/shell.js

```diff
@@ -1,14 +1,17 @@
 module.exports = {
+  ulimit: {
+    command: 'ulimit -n 9999'
+  },
   buildDist: {
     command: "./node_modules/.bin/browserify -g [ ./node_modules/reactify --extension '.jsx' ] ./index.js --standalone ui-toolkit > ./dist/ui-toolkit.standalone.js"
   },
   docsBuild: {
-    command: 'cd docs; npm run build; cd -',
+    command: 'cd docs; npm run build; cd -'
```

Comment:
Change to
    command: 'cd docs; npm run build && cd -'

To allow any errors from the build command to bubble up